README.md

```diff
@@ -6,7 +6,7 @@ Repo for Administrator
 2. Import this project as a maven project
 3. Set up MySQL schema using the schema in src/test/resources/schema.sql
 4. Create a file called `database-config.properties` in src/main/resources. Look at the java properties example down below for reference. *Note:*, you generally need `useSSL=false` in the database url. It is recommended to encrypt the connection (which is in Java)<br/>
-```
+```Java
```

Comment:
If this were actually Java code, `java` would be the correct tag; when I wrote my original comment, I couldn't figure out what the correct tag would be, but I've now done some testing and it looks like `java-properties` highlights the snippet the same way it highlights the code in the file in the repository.

Reply:
Oh wow, that is good to know. I will make that change.